client/blocks/app-banner/utils.js

```diff
@@ -43,12 +53,20 @@ export function getAppBannerData( translate, sectionName, isRTL ) {
 				copy: translate( 'See your real-time stats anytime, anywhere.' ),
 				icon: `/calypso/animations/app-promo/jp-stats${ isRTL ? '-rtl' : '' }.json`,
 			};
+		case HOME:
+			if ( displayJetpackAppBranding ) {
+				return {
+					title: translate( 'The Jetpack app makes WordPress better.' ),
+					copy: translate(
+						'Everything you need to write, publish, and manage a world-class site.'
+					),
+					icon: `/calypso/animations/app-promo/wp-to-jp${ isRTL ? '-rtl' : '' }.json`,
+				};
+			}
+
+			return emptyBanner;
```

Comment:
When I set `displayJetpackAppBranding` to false, I see that the banner is now showing on the home page, but with no content:

<img width="320" alt="Screenshot 2022-09-21 at 16 36 36" src="https://user-images.githubusercontent.com/2998162/191548467-e19f0ff4-d522-429e-992f-fe6f2088331f.png">

If there isn't a simple way to hide this in production for now, maybe we could update the default text to something more generic?

Reply:
Thank you for catching this. Addressed in 30b19ab.